lib/rich_text.rb

```diff
@@ -96,7 +136,7 @@ def to_text
 
   class Markdown < Base
     def to_html
-      linkify(sanitize(document.to_html), :all)
+      linkify(sanitize(document.to_html_bidi), :all)
```

Comment:
How does this work? I don't see anything defining a `to_html_bidi` method?

Reply:
https://kramdown.gettalong.org/rdoc/Kramdown/Parser/Base.html

> Implementing a new parser is rather easy: just derive a new class from this class and put it in the [Kramdown::Parser](https://kramdown.gettalong.org/rdoc/Kramdown/Parser.html) module – the latter is needed so that the auto-detection of the new parser works correctly.

Reply:
Oops, wrong link. This is the one: https://kramdown.gettalong.org/rdoc/Kramdown/Converter/Base.html

> Implementing a new converter is rather easy: just derive a new class from this class and put it in the [Kramdown::Converter](https://kramdown.gettalong.org/rdoc/Kramdown/Converter.html) module (the latter is only needed if auto-detection should work properly).

Basically the same thing.

kramdown converts "html_bidi" to "HtmlBidi" and picks the class using that.